srcpkgs/ghex/template

```diff
@@ -1,17 +1,25 @@
 # Template file for 'ghex'
 pkgname=ghex
-version=3.18.4
+version=42.2
 revision=1
 build_style=meson
-hostmakedepends="pkg-config gtk-update-icon-cache intltool itstool glib-devel"
-makedepends="gtk+3-devel iso-codes glib-devel"
+build_helper=gir
+configure_args="-Dintrospection=$(vopt_if gir enabled disabled)
+ $(vopt_bool gtk_doc gtk_doc)"
+hostmakedepends="pkg-config gtk4-devel gettext itstool glib-devel
+ desktop-file-utils $(vopt_if gtk_doc gi-docgen)"
+makedepends="gtk4-devel iso-codes glib-devel"
```

Comment:
* why is gtk4-devel required in $hostmakedepends?
* iso-codes can be moved to hostmakedepends
* glib-devel probably doesn't have to be in makedepends

Reply:
> why is gtk4-devel required in $hostmakedepends?

When building the documentation:
`ERROR: Could not find GIR dependency in the search paths: Gtk-4.0`
`gtk4-devel` also helps pull in `gtk-update-icon-cache`. If you want me to change it so that it uses gtk-update-icon-cache always and gtk4-devel or something else when it is building the docs, feel free to let me know.

> iso-codes can be moved to hostmakedepends

Alright

> glib-devel probably doesn't have to be in makedepends

Alright.

Reply:
> > why is gtk4-devel required in $hostmakedepends?
> 
> When building the documentation: `ERROR: Could not find GIR dependency in the search paths: Gtk-4.0` `gtk4-devel` also helps pull in `gtk-update-icon-cache`. If you want me to change it so that it uses gtk-update-icon-cache always and gtk4-devel or something else when it is building the docs, feel free to let me know.

makes sense